lms/djangoapps/ccx/plugins.py

```diff
@@ -31,6 +34,15 @@ def is_enabled(cls, course, user=None):
             # If ccx is not enable do not show ccx coach tab.
             return False
 
+        if not isinstance(course.id, CCXLocator):
+            if configuration_helpers.get_value('PCO_ENABLE_LICENSE_ENFORCEMENT', False) and not run_extension_point(
+                'PCO_IS_INSTITUTION_ADMIN_AND_LICENSE',
+                user=user,
+                master_course=course.id,
+            ):
+                # If course licensing is enable, then regular ccxs are disabled.
```

Comment:
Why don't you use just one `if` and check that the course is a CourseLocator instead?. @anfbermudezme